modules/persistent-storage-csi-vsphere-top-aware-overview.adoc

```diff
@@ -26,4 +26,20 @@ Datacenter: Atlanta
 Datacenter: Atlanta
 |openshift-region: us-east-1 (tag), openshift-zone: us-east-1b (tag)
 |This defines a different failure domain within the same region called us-east-1b.
-|===
\ No newline at end of file
+|===
+
+== vSphere CSI topology requirements 
+The following guidelines are recommended for vSphere CSI topology:
+
+* You are strongly recommended to add topology tags to datacenters and compute clusters, and *not* to hosts. 
++
+`vsphere-problem-detector` provides alerts if the `openshift-region` or `openshift-zone` tags are not defined at the datacenter or compute cluster level, and each topology tag (`openshift-region` or `openshift-zone`) should occur only once in the hierarchy.
++
+[NOTE]
+====
+While ignoring this advice only results in a log warning from the CSI driver and duplicate tags lower in the hierarchy, such as hosts, are ignored; VMware considers this an invalid configuration, and therefore to prevent problems you should not use it.
```

Comment:
Per ISG: Use _while_ only for a period of time.
```suggestion
Ignoring this advice only results in a log warning from the CSI driver and duplicate tags lower in the hierarchy, such as hosts, are ignored. VMware considers this an invalid configuration, and therefore to prevent problems you should not use it.
```